Makefile

```diff
@@ -39,7 +46,7 @@ docker-run:
 
 .PHONY: run-check
 run: check
-	go$(v) mod tidy; \
+	go mod tidy; \
```

Comment:
Are we removing this in the other component’s makefiles, too? With many of the other dependencies addressed, perhaps there’s no need to enforce golang version any longer. Is that the thought process?

Reply:
I need to add a check in place to make sure that the contributor who is forking the project to make sure that they are using Go 1.19

Reply:
Ideally, prior to a PR being opened otherwise they’ll face build check failures and have back and forth to deal with. Having this check in the makefile seems like the ideal spot - at dev/build time.